lib/libzfs/libzfs_status.c

```diff
@@ -51,27 +51,41 @@
 
 /*
  * Message ID table.  This must be kept in sync with the ZPOOL_STATUS_* defines
- * in libzfs.h.  Note that there are some status results which go past the end
- * of this table, and hence have no associated message ID.
+ * in include/libzfs.h.  Note that there are some status results which go past
+ * the end of this table, and hence have no associated message ID.
  */
 static char *zfs_msgid_table[] = {
-	"ZFS-8000-14",
-	"ZFS-8000-2Q",
-	"ZFS-8000-3C",
-	"ZFS-8000-4J",
-	"ZFS-8000-5E",
-	"ZFS-8000-6X",
-	"ZFS-8000-72",
-	"ZFS-8000-8A",
-	"ZFS-8000-9P",
-	"ZFS-8000-A5",
-	"ZFS-8000-EY",
-	"ZFS-8000-EY",
-	"ZFS-8000-EY",
-	"ZFS-8000-HC",
-	"ZFS-8000-JQ",
-	"ZFS-8000-K4",
-	"ZFS-8000-ER",
+	"ZFS-8000-14", /* CORRUPT_CACHE */
```

Comment:
nit: I think these would be more visible to new developers if your comment has the full name, e.g. **ZPOOL_STATUS_CORRUPT_CACHE**.  For example, I have looked for all instances of an existing **ZPOOL_STATUS_** macro name to look for places I need to add logic for a new macro I am introducing, and I tend to do that with git **grep -w**.

Reply:
I didn't think they would fit with the entire string, I can try though.

Reply:
Looks like it does fit after all, changed as suggested.